includes/admin/feedzy-rss-feeds-admin.php

```diff
@@ -205,6 +223,9 @@ public function enqueue_styles_admin() {
 		}
 
 		if ( 'feedzy_page_feedzy-support' === $screen->base || ( 'edit' !== $screen->base && 'feedzy_imports' === $screen->post_type ) ) {
+			if ( ! defined( 'CYPRESS_TESTING' ) ) {
```

Comment:
Could we move this check inside the `register_survey` function to have the code DRY ?